route/build/src/process_ntopo.f90

```diff
@@ -396,6 +397,7 @@ SUBROUTINE put_data_struct(nSeg, structSEG, structNTOPO, &
   character(len=strLen)                          :: cmessage         ! error message of downwind routine
   integer(i4b)                                   :: nUps             ! number of upstream segments for a segment
   integer(i4b)                                   :: iSeg,iUps        ! loop indices
+  integer(i4b), parameter                        :: doll=1
```

Comment:
Lake model type. If natural lake, it's doll.

This could use a comment to explain.